source/change-streams.rst

```diff
@@ -201,21 +201,25 @@ Driver API
 
   interface ChangeStream extends Iterable<Document> {
     /**
+     * @since 3.6
```

Comment:
Is this redundant given that the spec's own header states "Minimum Server Version: 3.6"? I'd assume that all APIs effectively default to `@since 3.6` unless otherwise stated.

Reply:
done